app/views/partials/_header-global.html.erb

```diff
@@ -0,0 +1,6 @@
+<header class="global-header">
+  <h1>You vs. The Machine</h1>
+  <p>The age-old struggle ends now.
+  Choose from the list of the problems,
+  and we'll supply the hacks to defeat your machine.</p>
+</header>
```

Comment:
This file should be moved under the shared directory. You can have a subdirectory differentiating it from the svg partials if you think it's needed.